user_docs/en/changes.t2t

```diff
@@ -127,7 +127,8 @@ Users of Poedit 1 are encouraged to update to Poedit 3 if they want to rely on e
   - This new behavior can be disabled using the new "Use enhanced event processing" setting in NVDA's advanced settings.
   -
 - NVDA is again able to track the focus in applications running within Windows Defender Application Guard (WDAG). (#15164)
-- The speech text is no longer updated when the mouse moves in the Speech Viewer. (#15953, @hwf1324)
+- The speech text is no longer updated when the mouse moves in the Speech Viewer. (#15952, @hwf1324)
+- Fix the error of playing audio coordinates while the application is in sleep mode when the play mouse audio coordinates feature is enabled. (#8059, hwf1324)

```

Comment:
this PR doesn't target the 2024.1 beta and it shouldn't. Please merge in the latest master and move the changes to the 2024.2 change log

Reply:
Okay, can you help me fix the issue with the incorrect PR number in my previous PR when organizing `user_docs/en/changes.t2t` for 2024.1 beta?

Reply:
please open a separate PR to beta using a branch based on beta to fix this